### DISCUSSION TITLE
Dining Philosophers execise solution in 67.3 is Livelock.Why not use the same idea of 62.3?

### DISCUSSION THREAD

**Discussion by xiaohu-zhang** [google/comprehensive-rust#2367] [General]
Dining Philosophers execise solution in 67.3 is Livelock.Why not use the same idea of 62.3?

**Comment by djmitche** [comment]
I suspect that's just an oversight. Do you want to make a PR to fix it?

**Reply by djmitche** [reply]
Filed https://github.com/google/comprehensive-rust/issues/2371 to track this.

**Reply by djmitche** [reply]
Oops, #2368 covers this.

**Reply by xiaohu-zhang** [reply]
no problem.I will pull a pr to fix it.
